src/hyperlight_common/src/flatbuffer_wrappers/function_call.rs

```diff
@@ -288,9 +223,40 @@ impl TryFrom<FunctionCall> for Vec<u8> {
             },
         );
         builder.finish_size_prefixed(function_call, None);
-        let res = builder.finished_data().to_vec();
```

Comment:
This `to_vec` is the clone I was able to avoid, which is the most important part of this PR